pulse/inngest-router/README.md

```diff
@@ -0,0 +1,125 @@
+# Prisma Pulse-Inngest Router
+
+This package enables event-driven support for your existing Inngest workflows, using Prisma Pulse. Trigger workflows instantly whenever a database table changes, so you can create timely workflows that run instantly -- not only batch workflows.
+
+## Pre-requisites
+
+- An existing project with [Inngest](https://www.inngest.com/docs) and [Prisma Client](https://www.prisma.io/docs/getting-started) configured
+- A **Pulse API key** which you can get by enabling Pulse in a project in your
+  [Prisma Data Platform](https://pris.ly/pdp) account (learn more in the
+  [docs](https://www.prisma.io/docs/platform/concepts/environments#api-keys))
+- An existing [Inngest](https://www.inngest.com) account and
+  [Event Key](https://www.inngest.com/docs/events/creating-an-event-key?ref=environment-variables)
+
+## Get started
+
+### Fork or clone this repo
+
+```bash
+npx try-prisma@latest --template pulse/inngest-router
+```
+
+You should operate with your own version of this service in a GitHub repository that you own. This allows you to track changes, and deploy to cloud platforms easily.
+
+### Set up your API keys
+
+Make a copy of the `.env.example` file, rename it to `.env`, and update it with your own keys and database connection string:
+
+```bash
+cp .env.example .env
+```
+
+### Add your Prisma schema
+
+Add your existing Prisma schema file to the `/prisma` directory, overwriting the existing example schema.
+
+If you don't already have a project using Inngest and Prisma Client, you may choose to try another [Prisma example](https://github.com/prisma/prisma-examples) instead.
+
+### Define the Prisma models to subscribe to
+
+Add all Prisma models that you wish to stream events from to the array. The service will ignore any models that don't exist.
+
+Note that the naming convention should follow the standard naming convention for Prisma ORM, eg. `MyModel` becomes `myModel`.
+
+```typescript
+// Here configure each prisma model to stream changes from
+const PRISMA_MODELS = ['notification', 'user'];
+```
+
+### Test locally before saving your changes
+
+Run the service locally:
+1. `npx prisma generate`
+1. `npm run build`
+1. `npm run start`
+
+**Hint:** you can trigger changes in your database by running
+`npx prisma studio` and modifying some data or adding a new row.
+
+### Commit your changes
+
+```
+git add .
+git commit -m "Configure my pulse-inngest-router service"
+git push
+```
+
+## Deploy
+
+### To Railway
+
+1. Create an account and project on [Railway](https://railway.app/new)
+1. Create a new service by clicking the **Create** button in the top right of
+your dashboard
+1. Choose to deploy from a **GitHub Repo**
+1. Authorize your GitHub account, and choose the GitHub repository you used in
+previous steps
+1. Add the respective environment variables:
+    1. DATABASE_URL
+    1. PULSE_API_KEY
+    1. INNGEST_EVENT_KEY
```

Comment:
Would change these to bullets instead of numbers (they don't have an inherent order)
```suggestion
   * DATABASE_URL
   * PULSE_API_KEY
   * INNGEST_EVENT_KEY
```